include/LightGBM/dataset.h

```diff
@@ -13,6 +13,8 @@
 #include <LightGBM/utils/random.h>
 #include <LightGBM/utils/text_reader.h>
 
+// #include "../../src/io/parser.hpp"
```

Comment:
>/ #include "../../src/io/parser.hpp" [](http://example.com/codeflow?start=1&length=36)

if you don't need this line, you can simply remove it instead of commenting it. :-)